NDTensors/ext/NDTensorsCUDAExt/set_types.jl

```diff
@@ -38,3 +38,18 @@ default_parameter(::Type{<:CuArray}, ::Position{3}) = Mem.DeviceBuffer
 nparameters(::Type{<:CuArray}) = Val(3)
 
 SetParameters.unspecify_parameters(::Type{<:CuArray}) = CuArray
+
+using NDTensors.TypeParameterAccessors: TypeParameterAccessors
+function TypeParameterAccessors.position(::Type{<:CuArray}, ::typeof(eltype))
+  return TypeParameterAccessors.Position(1)
```

Comment:
```suggestion
  return Position(1)
```
and add `Position` to the `using` statement.

Reply:
Okay I'll make sure to do that next time, I also added comments in the code